src/melee/gr/gricemt.c

```diff
@@ -683,7 +687,19 @@ void grIceMt_801F87C8(Ground_GObj* param1)
     return;
 }
 
-/// #grIceMt_801F87FC
+/// @todo Rename: This is callback3 (destroy) for row 5 in grIm_803E4718.
+/// Destroys Item_GObj* stored in icemt.xF4[1-5].
+void grIceMt_801F87FC(Ground_GObj* gobj)
+{
+    int i;
+    Ground* gp = GET_GROUND(gobj);
+    for (i = 0; i < 5; i++) {
+        HSD_GObj* temp = gp->gv.icemt.xF4[i + 1];
```

Comment:
See my other comment, if grIceMt_GroundVars is changed accordingly, this would be simplified to `gp->gv.icemt.xF4[i];`. Haven't tested that but I wouldn't expect it to change codegen.